models/web/sources.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 sources:
   - name: atomic
     schema: "{{ var('snowplow__atomic_schema', 'atomic') }}"
-    database: "{{ var('snowplow__database', target.database) }}"
+    database: "{{ var('snowplow__database', target.database) if target.type not in ['databricks', 'spark'] else var('snowplow__atomic_schema', 'atomic') }}"
```

Comment:
This we might need to change based on databricks connector v1.1.1 adding support for the unity catalog. Could just remove it actually and then use the `hive_metastore` as a database value for Databricks I think